src/components/Sources.js

```diff
@@ -18,13 +19,21 @@ const Sources = React.createClass({
 
   displayName: "Sources",
 
+  header() {
+    if (isApplication()) {
+      return "Firefox Debugger";
+    }
+
+    return L10N.getStr("sources.header");
+  },
+
```

Comment:
Not fully convinced we need this. It seems like a special case that diverges from the design for not much reason. Also, it's confusing if you are debugging node and it says "Firefox Debugger".